### DISCUSSION TITLE
Any plan to add support for Gore detection?

### DISCUSSION THREAD

**Discussion by nomadshiba** [infinitered/nsfwjs#711] [Ideas]
Any plan to add support for Gore detection?
Would be useful, because if you dont want nsfw stuff you probably dont want gore too

**Comment by GantMan** [comment]
Currently no, but I like that idea.   I am short on free-time to build a new data category, but that's a great suggestion.

I'm going to add this to a list of (wouldn't that be amazing) for future development.   Because this project is not funded, it's slow to add new features.

**Comment by moraxh** [comment]
Any news?
